packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -161,17 +177,8 @@ export interface NextjsSiteProps extends Omit<SsrSiteProps, "nodejs"> {
   };
 }
 
-const LAYER_VERSION = "2";
-const DEFAULT_OPEN_NEXT_VERSION = "2.3.8";
-const DEFAULT_CACHE_POLICY_ALLOWED_HEADERS = [
-  "accept",
-  "rsc",
-  "next-router-prefetch",
-  "next-router-state-tree",
-  "next-url",
-  "x-prerender-bypass",
-  "x-prerender-revalidate"
-];
+const DEFAULT_OPEN_NEXT_VERSION = "3.0.0-rc.15";
```

Comment:
I forgot about that, but we should bump this one to 3.0.1 before final release

Reply:
yup!